django/db/models/base.py

```diff
@@ -44,6 +45,20 @@ def __str__(self):
 DEFERRED = Deferred()
 
 
+class ExceptionMeta(type):
+    """
+    Custom metaclass for dynamically created model exceptions
+    """
+    pass
+
+
+def pickle_exc_meta(obj):
+    return getattr, (obj.model, 'DoesNotExist')
```

Comment:
Maybe use `obj.__name__` instead of `DoesNotExist`?

Reply:
Good call - this works.